pkg/trace/obfuscate/sql_tokenizer.go

```diff
@@ -248,7 +272,7 @@ func (tkn *SQLTokenizer) scanIdentifier() (TokenKind, []byte) {
 	buffer.WriteRune(tkn.lastChar)
 	tkn.next()
 
-	for isLetter(tkn.lastChar) || isDigit(tkn.lastChar) || tkn.lastChar == '.' || tkn.lastChar == '*' {
+	for isLetter(tkn.lastChar) || isDigit(tkn.lastChar) || tkn.lastChar == '*' {
```

Comment:
I am not sure the full extent of this change. Are there any non-literal identifiers in any engine which have periods in them?